lib/web/apiserver_test.go

```diff
@@ -1144,7 +1152,7 @@ t1ready:
 	for {
 		select {
 		case <-done:
-			require.FailNow(t, "expected to receive 2 resize events (got %d) and at least 1 raw event (got %d)", t1ResizeEvents, t1RawEvents)
+			require.FailNow(t, fmt.Sprintf("expected to receive 2 resize events (got %d) and at least 1 raw event (got %d)", t1ResizeEvents, t1RawEvents))
```

Comment:
```suggestion
			require.FailNow(t, "expected to receive 2 resize events (got %d) and at least 1 raw event (got %d)", t1ResizeEvents, t1RawEvents)
```

FailNow already behaves like sprintf.

Reply:
updated to use `require.FailNowf`